Source/MQTTnet/Server/MqttServer.cs

```diff
@@ -190,6 +190,8 @@ public Task ClearRetainedApplicationMessagesAsync()
             return _retainedMessagesManager?.ClearMessagesAsync();
         }
 
+        public bool IsStarted => _cancellationTokenSource != null;
```

Comment:
Please also move this to the server interface because you have to cast the object from the MQTT factory.

Reply:
Also please move the property to the property section of the file. Be best is at corresponding other properties for a similar purpose.

And please merge master and fix conflicts.